source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,125 @@
+.. _comet-lake-rvp:

```

Comment:
There are couple of warnings during the HTML compiling.
EX:
   WARNING: Undefined substitution referenced: "Comet Lake S", "CML", "H", "V", "H420E", etc.   
Please ensure it can pass HTML build without warning by running "make html"

Reply:
I have added those to substitutions.txt. Currently there is no more warning when running "make html"